doc/changes/changes_0.5.0.md

```diff
@@ -1,11 +1,13 @@
 # row-level-security-lua 0.5.0, released 2021-09-01
 
-Code name: Support for Default SRID
+Code name: Schema refresh, exclude capabilities and default SRID
 
 ## Summary
 
 Version 0.5.0 of `row-level-security` adds support for `GEOMETRY` columns without explicit SRID.
 
+Lua RLS now supports `ALTER VIRTUAL SCHEMA ... REFRESH` to update the metadata of an existing virtual schema.
```

Comment:
Is the `RLS` abbreviation well known outside Exasol?

Reply:
Yes, but I wrote it out anyway. Just in case.